examples/plot_conditional_vs_marginal_xor_data.py

```diff
@@ -0,0 +1,143 @@
+"""
+Conditional vs Marginal Importance on the XOR dataset
+==================================================================
+
+This example illustrates on XOR data that variables can be conditionally important even
+if they are not marginally important. The conditional importance is computed using the
+CPI class and the marginal importance is computed using univariate models.
+"""
+
+#############################################################################
```

Comment:
Can you put this line after the import?

In my view, it's better to have a cell with just the import.